### PR TITLE
API-10240 add support for list_channels and get_product_limits_for_plan

### PR DESCRIPTION
**Pull request by MaciejSzaflik**
Added support for list_channels and check_product_limits_for_plan
go sdk -> https://github.com/livechat/lc-sdk-go/pull/104

### CODE REVIEW

File: src/configuration/structures.ts [hunk 1]
```diff
@@ -202,3 +202,15 @@ export interface GetOrganizationIDResponse {
 export interface GetLicenseIDResponse {
   license_id: string;
 }
+
+export interface PlanLimit {
+	resource:     string;
+	limit_balance: number;
+	id?:        string;
```
Comment (grz-gajda) [inline src/configuration/structures.ts:209]: Please fix the formatting.

File: src/configuration/index.ts [hunk 2]
```diff
@@ -430,4 +432,19 @@ export default class ConfigurationAPI extends WebAPI {
   async getLicenseID(organizationID: string): Promise<GetLicenseIDResponse> {
     return this.send("get_license_id", { organization_id: organizationID });
   }
+
+  /**
+   * Compares your organization's current resources with a given plan and returns those which exceeded the called plan's limits.
+   * @param plan - `plan` must be one of: 'starter', 'team', 'enterprise' or 'enterpriseplus'
+   */
+   async checkProductLimitsForPlan(plan: string): Promise<PlanLimit[]> {
```
Comment (grz-gajda) [inline src/configuration/index.ts:440]: Please fix the formatting.

**Review by grz-gajda — APPROVED** [review]

File: src/configuration/index.ts (hunk 2 above)
Comment (grz-gajda) [inline src/configuration/index.ts:440]: Please fix the formatting.

**Review by grz-gajda — CHANGES_REQUESTED** [review]

**Comment by grz-gajda** [comment]
The first approve was the missclick 👀  .

File: src/configuration/structures.ts (hunk 1 above)
Comment (MaciejSzaflik) [reply src/configuration/structures.ts]: tabs removed, spaces removed

File: src/configuration/index.ts (hunk 2 above)
Comment (MaciejSzaflik) [reply src/configuration/index.ts]: space removed

**Review by grz-gajda — APPROVED** [review]

**Pull request merged**